### DISCUSSION TITLE
Scrcpy 2.0 Audio Preview

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#536] [Q&A] [answered]
hey, I tested the preview at #495, but I couldn't seem to get audio to output to my computer, but the audio stopped on my phone. is it only for recording, or is there something that hasn't been completed yet, or do I need to enable or change a setting?

**Comment by CorruptionVR** [comment]
quick update, the raw audio codec breaks things on my chromebook. will test on windows 10.

edit: breaks on windows 10 as well.

**Comment by CorruptionVR** [comment, marked as answer]
nevermind. looks like that it's just quiet, and that the audio playing icon doesn't show on the tab. but for the audio, It would be nice if there was a switch to either enable or disable the audio socket.

**Reply by yume-chan** [reply]
Option for disabling audio added.

**Reply by CorruptionVR** [reply]
hey, did you mean to remove the settings flyout? because, uhhhhh
![image](https://user-images.githubusercontent.com/78228404/231544955-4768028b-f31a-4c0d-9392-b42a37dd909e.png)
![image](https://user-images.githubusercontent.com/78228404/231545194-4be93e3b-a32b-4774-bac9-1c0be1cb5b9a.png)
the settings button is gone, and only shows when scrcpy is stopped.
edit - is this in preparation for the popup display?

**Reply by yume-chan** [reply]
>  is this in preparation for the popup display?

Yes
